From: =?utf-8?q?IOhannes_m_zm=C3=B6lnig?= <umlaeute@debian.org>
Date: Thu, 9 Nov 2017 00:00:00 +0100
Subject: prevent Projucer from checking for updates

 The check-for-updates only makes sense for user-installed Projucer, not for
 the Debian packaged version.
Forwarded: not-needed
---
 extras/Projucer/Source/Application/jucer_AutoUpdater.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/extras/Projucer/Source/Application/jucer_AutoUpdater.cpp b/extras/Projucer/Source/Application/jucer_AutoUpdater.cpp
index a54b527..2014b10 100644
--- a/extras/Projucer/Source/Application/jucer_AutoUpdater.cpp
+++ b/extras/Projucer/Source/Application/jucer_AutoUpdater.cpp
@@ -42,6 +42,7 @@ LatestVersionCheckerAndUpdater::~LatestVersionCheckerAndUpdater()
 
 void LatestVersionCheckerAndUpdater::checkForNewVersion (bool showAlerts)
 {
+    return; // DONT check for new versions on Debian
     if (! isThreadRunning())
     {
         showAlertWindows = showAlerts;
